CMakeLists.txt

```diff
@@ -183,6 +195,11 @@ if(NOT NUMA_LIBRARY)
     message(FATAL_ERROR "NUMA library not found")
 endif()
 
+find_package(umd)
```

Comment:
Hmm, why is this needed if these are included by dependencies down below (ie umd)? Is there something I'm missing?

Reply:
cmake 3.22.1 complains that it can't find the `umd` package. explicitly adding that line corrected the issue.

Reply:
I don't get how find_package is working with CPM here. We don't normally need it.

Reply:
It doesn't make sense to me why this would be needed, unless the workflow that needed it was to not use submodules form tt-metal and instead use system libs.  But this build would fail if the submodules were absent.  Only other theory is that this branch was originally based on something old and the situation was different back then :)

Our CI system is using CMake v3.22.2.  I've kicked off a run based on this branch but with a commit that removes all of these `find_package` calls.  Pending results here: https://github.com/tenstorrent/tt-metal/actions/runs/12073412996
That run will, naturally, be for x86.  @ct-clmsn can you share what error message you see with the `find_package` calls absent now that this branch is based on a more recent version of tt-metal?  That'll help us understand what's different.

Reply:
@afuller-TT I doubled back and remembered this was a consequence of building tt-umd out of the tt-metal directory. The latest tt-umd has riscv support but that branch hasn't been pulled into tt-metal as of yet. I'll remove the `find_package` as doing a symbolic link gets me around this issue. apologies for the confusion.